### PR TITLE
Create a default security policy

### PR DESCRIPTION
**Pull request by bjgill**
This security policy will apply to all repos that haven't set their own policy - see https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file.

This should make it much easier to manage our security policy. We have over 100 repos in our organisation, and having to set the security policy for each individually would be really rather time-consuming.

This is a copy of the authoritative version maintained by @ramya-chan and co.: https://github.com/Crown-Commercial-Service/ccs-vulnerability-disclosure-policy/blob/main/security.txt

### CODE REVIEW

**Comment by bjgill** [comment]
@ramya-chan - any objections to setting this as the default security policy for all repos in the CCS organisation?

Once this is merged, I'll be able to remove all the security policies from the Digital Marketplace repos that have the wrong URLs and rely on this correct default.

**Comment by ramya-chan** [comment]
@bjgill No, I do not have any objections creating a default security policy for smarter execution. 

https://github.com/Crown-Commercial-Service/ccs-vulnerability-disclosure-policy/blob/main/security.txt

This is our CCS Vulnerability Disclosure Repo, maintained by the security team, so this is the parent link, as any changes will be made here(so it does not require changes elsewhere), kindly reference this. 

Objective: As part of NCSC Minimum Cyber Security Standards, all repos should have a security.txt file for security researchers (that is what they will be looking for) who may want to report any vulnerabilities in our assets. So you have the security.txt file or link to the file is absolutely fine.

**Comment by bjgill** [comment]
@ramya-chan - thanks. Replaced the content with a link to the authoritative version. This look sensible?

**Comment by ramya-chan** [comment]
@bjgill sorry the previous looked fine, I didnt mean you to replace contents with the link.

**Comment by bjgill** [comment]
> @bjgill sorry the previous looked fine, I didnt mean you to replace contents with the link.

Oh, sorry. Misunderstood you. I'll revert.

I'll then merge it by CoB today unless anyone from the rest of the dev team objects.

**Review by tberey — APPROVED** [review]

**Pull request merged**
